tools/build_variables.bzl

```diff
@@ -353,9 +353,8 @@ libtorch_extra_sources = libtorch_core_jit_sources + [
 def libtorch_sources(gencode_pattern = ":generate-code[{}]"):
     return libtorch_generated_sources(gencode_pattern) + libtorch_core_sources + libtorch_distributed_sources + libtorch_extra_sources
 
-libtorch_cuda_sources = [
+libtorch_cuda_core_sources = [
     "torch/csrc/cuda/comm.cpp",
```

Comment:
nit should we move this to libtorch_cuda_sources as well? (doesn't seem to be used anyway)